### PR TITLE
Get all data into one place

### LINKED ISSUE #2: combine ABP, ILSFA and utility data sources
To get an understanding of all the solar projects in Illinois, we need to pull from 3 sources:

* Illinois' Adjustable Block Program: `data/v2/raw/report-3-census-tract-rev.csv`
* Illinois Solar for All: `data/v2/raw/ilsfa-2023-05-07.csv`
* Utility Scale solar: `data/v2/raw/solar-eia-plants_1677797680150.geojson`

These files need to be combined into one data source.

### PR DESCRIPTION
**Pull request by SuragNuthulapaty**
Combined data from 
`report-3-census-tract-rev.csv`
`ilsfa-2023-05-07.csv`
`solar-eia-plants_1677797680150.geojson`
all into one dictionary, and then aggregated data based upon that

closes #2

### CODE REVIEW

File: data/v2/test.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import requests
```
Comment (derekeder) [inline data/v2/test.py:1]: we don't need to commit this file

File: data/v2/aggregate-data-by-county.py [hunk 2]
```diff
@@ -1,38 +1,102 @@
 import csv
 import json
+import requests
+import requests_cache
```
Comment (derekeder) [inline data/v2/aggregate-data-by-county.py:4]: lets add requests and requests-cache to requirements.txt

**Review by derekeder — CHANGES_REQUESTED** [review]
looks good - just a few smaller changes to make and we can bring this in

**Comment by SuragNuthulapaty** [comment]
@derekeder I changed the requested changes and updated the readme

**Review by derekeder — APPROVED** [review]
thanks - looks good!

**Pull request merged**